spec/dill/widgets/widget_spec.rb

```diff
@@ -211,46 +211,36 @@ class MyWidget < Dill::Widget
       end
     end
 
-    describe '#inspect', if: driver == :webkit do
+    describe '#html', if: driver == :webkit do
       GivenHTML <<-HTML
-        <span id="ins">Ins</span>
+        <span id="html" class="one">HTML</span>
       HTML
 
       GivenWidget do
         class MyWidget < Dill::Widget
           root 'span'
-
-          def self.name
-            'Inspect'
-          end
         end
       end
 
-      When(:inspection) { widget(:my_widget).inspect }
+      When(:html) { widget(:my_widget).html }
 
-      Then { inspection == "<!-- Inspect: -->\n<span id=\"ins\">Ins</span>\n" }
+      Then { html == "<span id=\"html\" class=\"one\">HTML</span>" }
```

Comment:
@aesnyder I stripped the newline, but when we output a string with double quotes, I think we're stuck with escaping them.